CMakeLists.txt

```diff
@@ -835,32 +836,18 @@ set (HDF5_SRC_INCLUDE_DIRS
     ${HDF5_SRC_INCLUDE_DIRS}
     ${H5FD_SUBFILING_DIR}
 )
-option (HDF5_ENABLE_SUBFILING_VFD "Build Parallel HDF5 Subfiling VFD" OFF)
+cmake_dependent_option(HDF5_ENABLE_SUBFILING_VFD "Build Parallel HDF5 Subfiling VFD" ON "HDF5_ENABLE_PARALLEL AND NOT WIN32" OFF)
```

Comment:
Also, it looks like this syntax for the condition is only supported as of CMake 3.22 and otherwise has to be a semicolon-separated list.

Reply:
Since we are still on 3.18 then we need to use the semis